lib/saml/user_attributes/ssoe.rb

```diff
@@ -16,9 +16,10 @@ class SSOe
 
       attr_reader :attributes, :authn_context, :warnings
 
-      def initialize(saml_attributes, authn_context)
+      def initialize(saml_attributes, authn_context, saml_settings = nil)
```

Comment:
Ah, I was gonna say we need to make this change to all of the user attributes classes, but it looks like doing it here and in base takes care of that!